src/pipelines/encore/encore-pipeline.ts

```diff
@@ -0,0 +1,343 @@
+import { Resolution } from '../../models/resolution';
+import { Pipeline } from '../pipeline';
+import AWSPipeline from '../aws/aws-pipeline';
+import { AWSPipelineConfiguration } from '../aws/aws-pipeline-configuration';
+import { EncorePipelineConfiguration } from './encore-pipeline-configuration';
+import { QualityAnalysisModel } from '../../models/quality-analysis-model';
+import { EncoreInstance } from '../../models/encoreInstance';
+import { EncoreJobs, EncoreJob } from '../../models/encoreJobs';
+import logger from '../../logger';
+import fs from 'fs';
+const ISOBoxer = require('codem-isoboxer');
+const { Readable } = require('stream'); //Typescript import library contains different functions. 
+//If someone knows how to achieve the same functionality in Typescript syntax let me know.
+const { finished } = require('stream/promises'); //Same as above.
+
+export function delay(ms: number): Promise<void> {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+export class EncorePipeline implements Pipeline {
+
+  configuration: EncorePipelineConfiguration;
+
+  constructor(configuration: EncorePipelineConfiguration) {
+    this.configuration = configuration;
+  }
+
+  async transcode(input: string, targetResolution: Resolution, targetBitrate: number, output: string, variables?: Record<string, string>): Promise<string> {
+    return "todo"
+  }
```

Comment:
Maybe we should throw an exception here just to make sure that it doesn't fail silently?

Reply:
This function will be unused. How should we handle it?

Reply:
See https://github.com/Eyevinn/autovmaf/pull/58#discussion_r1365242726